File: Collector/OLD/LegacyJava/Collector/Restrictions.java
package Collector;

public interface Restrictions {
    int SEED = 1;

    float VIEWPORT_HEIGHT = 9/2f;
    float VIEWPORT_WIDTH = 16/2f;

    int MOVEMENT_SPEED = 16;
    int FREE_SPEED = 2;
    int KEY_DELAY = 0;

    int TILE_SIZE = 16;
    int CHUNK_SIZE = 8;
    int SUPER_CHUNK_SIZE = 1;

    int TILE_SHIFT = (int)(Math.log(TILE_SIZE)/Math.log(2));
    int CHUNK_SHIFT = (int)(Math.log(CHUNK_SIZE)/Math.log(2));
    //int SUPER_CHUNK_SHIFT = (int)(Math.log(SUPER_CHUNK_SIZE)/Math.log(2));

    int RENDER_DISTANCE = 3;
    float RENDER_TIME = 1/60f;
}

File: Collector/Restrictions.cs
using System;
using Collector.Character;
using Microsoft.Xna.Framework;
using Microsoft.Xna.Framework.Graphics;
using static Collector.IRestrictions;

namespace Collector
{
    public interface IRestrictions
    {
        public const int Seed = 3;
        public const int TileSize = 1;
        public const float ViewportHeight = 9f;
        public const float ViewportWidth = 16f;
        public const float MovementSpeed = 16/32f;
        public const int KeyDelay = 20;
        public const int ChunkSize = 8;
        public const int SuperChunkSize = 1;
        public const int ChunkShift = 3;
        public const int RenderDistance = 5;
        public const float RenderTime = 1/60f;
        public const float Zoom = 69;
        public const float Scale = ViewportWidth/ViewportHeight;
        public const float AnimationDuration = 0.3f;
    }
}